### DISCUSSION TITLE
Any update on how soon we will see the new DynamoDb integration feature?

### DISCUSSION THREAD

**Discussion by martinpsz** [pulumi/pulumi#15042] [General]
Hi,

I see that AWS announcing Integrations at ReInvent for DynamoDB. I am interested in having this option, which would allow for direct connection between a Dynamo stream and the OpenSearch service. Any idea on when this will show up in Pulumi AWS Native?

**Comment by Firdaus-prog** [comment]
Can we open it back, Im also figuring out how to do zero-ETL OpenSearch with Dynamodb
